### DISCUSSION TITLE
Add skip link to main content

### DISCUSSION THREAD

**Discussion by trallard** [pradyunsg/furo#741] [Ideas] [answered]
While navigating/reading the [Python devguide](https://devguide.python.org/), I realised this theme does not have a skip link to help readers who do not use a mouse or pointing devices quickly navigate to the main content on the page. 

This is required to meet [WCAG criteria 2.4.1](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks). I would be happy to contribute this feature to the theme if you are ok with it.

**Comment by trallard** [comment, marked as answer]
Noticed there is already an issue 🤦🏽‍♀️  and I missed it. Will close this, sorry.

**Reply by pradyunsg** [reply]
For posterity: https://github.com/pradyunsg/furo/issues/440
